styles/toolbar.css

```diff
@@ -1,8 +1,8 @@
 .h5p-dragnbar {
-  height: 7%;
+  height: 1.5em;
```

Comment:
I assume this is done to prevent the buttons changing size when changing the size of the course presentation itself?

Reply:
jepp